Extension/src/LanguageServer/extension.ts

```diff
@@ -254,6 +262,20 @@ export async function activate(): Promise<void> {
         const tool = vscode.lm.registerTool('cpptools-lmtool-configuration', new CppConfigurationLanguageModelTool());
         disposables.push(tool);
     }
+
+    if (isRelatedFilesApiEnabled) {
+        const copilotExtension = vscode.extensions.getExtension<CopilotApi>('github.copilot');
+        if (util.extensionContext && copilotExtension) {
+            const api: CopilotApi = copilotExtension.isActive ? copilotExtension.exports : await copilotExtension.activate();
```

Comment:
In our API's helper, we catch exceptions which can be thrown during `activate()`. You should do that too.

https://github.com/microsoft/vscode-cpptools-api/blob/5eff4c0ff7ad41ef58ba4e98bae5cc8420d2a15c/api.ts#L334C9-L340C10